ui/public/locales/en.json

```diff
@@ -444,9 +444,9 @@
 "label.annotated.by": "Annotator",
 "label.annotation": "Comment",
 "label.annotations": "Comments",
-"label.annotation.admins.only": "Only visible to Administrators",
```

Comment:
Please check the usage of these keys - they break the comments feature UI:
<img width="540" alt="Screen Shot 2022-02-22 at 11 32 41" src="https://user-images.githubusercontent.com/5295080/155153650-2510896e-f810-43b3-883c-de9d10dc2dd7.png">

Reply:
Thanks @nvazquez , I've updated it.

Reply:
Thanks